weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/LLMDropdownOptions.tsx

```diff
@@ -0,0 +1,286 @@
+import {Box} from '@mui/material';
+import {
+  MOON_100,
+  MOON_800,
+  MOON_900,
+  OBLIVION,
+  TEAL_500,
+} from '@wandb/weave/common/css/color.styles';
+import {hexToRGB} from '@wandb/weave/common/css/utils';
+import {Icon} from '@wandb/weave/components/Icon';
+import {Tooltip} from '@wandb/weave/components/Tooltip';
+import React, {useEffect, useRef, useState} from 'react';
+import ReactDOM from 'react-dom';
+import {components, OptionProps} from 'react-select';
+
+import {Link} from '../../common/Links';
+import {LLMMaxTokensKey} from '../llmMaxTokens';
+
+export interface ProviderOption {
+  label: string | React.ReactNode;
+  value: string;
+  llms: Array<{
+    label: string;
+    value: LLMMaxTokensKey;
+    max_tokens: number;
+  }>;
+  isDisabled?: boolean;
+}
+
+export interface CustomOptionProps extends OptionProps<ProviderOption, false> {
+  onChange: (value: LLMMaxTokensKey, maxTokens: number) => void;
+  entity: string;
+  project: string;
+}
+
+export const DisabledProviderTooltip: React.FC<{
+  children: React.ReactNode;
+  entity: string;
+}> = ({children, entity}) => {
+  return (
+    <Tooltip
+      trigger={children}
+      content={
+        <Box
+          sx={{
+            backgroundColor: MOON_900,
+            color: 'white',
+            fontSize: '14px',
+            fontWeight: 400,
+            borderRadius: '4px',
+          }}>
+          <div>This provider is not configured.</div>
+          <div>
+            Check{' '}
+            <Link
+              to={`/${entity}/settings`}
```

Comment:
I think this link won't work for personal entities or if you aren't an admin in the team entity.